README.md

```diff
@@ -114,7 +110,7 @@ You can also use `setHeaders()` and `addRow` with Arrays.
 
 ## Example 5: Non-bordered Table without Header
 
-    require_once 'ConsoleTable.php';
+    require "vendor/autoload.php";
```

Comment:
Can you please remove this line since it is only valid for composer users? We shall not include `require` in all examples.